images/tf-serve-gpu/Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM cortexlabs/tf-base-gpu
 
-ARG TF_VERSION="1.12.0"
+ARG TF_SERV_VERSION="1.12.0"
```

Comment:
I went to their [releases](https://github.com/tensorflow/serving/releases) and noticed that they have `1.13.0`, so I tried that and it worked. Can you update to `1.13.0`?